src/models/document/document-content-tests/dc-general.test.ts

```diff
@@ -44,16 +44,16 @@ describe("DocumentContentModel", () => {
 
   it("allows the tool tiles to be added", () => {
     expect(documentContent.tileMap.size).toBe(0);
-    documentContent.addTile("text");
+    documentContent.addTile("text", { title: "Text 1" });
```

Comment:
A note on all these changes: addTile no longer sets a default title. The caller is responsible to setting this, since there are different scenarios where the title should be different things (or left empty).

Reply:
After our discussion, I moved the generation of a unique title for toolbar-added tiles from `toolbar.tsx` into `document.ts`.  This doesn't help with these changes to the DocumentContent tests, but does give us a clean entry point for tests where we have a `Document` handy.  I added such a test in `document.test.ts`.